pkg/utils/utils.go

```diff
@@ -4,8 +4,11 @@ import (
 	"context"
 	"math"
 	mrand "math/rand"
+	"sync"
 	"time"
 
+	"github.com/pkg/errors"
```

Comment:
We're trying to stick to std lib errors:
```suggestion
	"errors"
```